### PR TITLE
Css

### PR DESCRIPTION
**Pull request by gomezbiggeri**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
Worked on CSS and JSX files to match Figma designs and remove some old legacy features.

- Describe what changes are being made
Changes to the login page, footer, admin Calendar page, and upload modal. 

- Describe why these changes are being made
UI/UX improvements

- List the use cases and edge cases relevant to this PR
N/A

- Describe how errors will be handled. How will we know if this code breaks in production
N/A

- Describe any external libraries/dependencies added or removed in this PR
N/A

- Describe any security risks are there regarding this change
N/A

- Describe how you tested these changes
Docker local environment

- Link to relevant external documentation
N/A

--------------------------
### :clipboard: Mandatory Checklist

- [x] Example of a checked item (please remove when creating your Pull Request) 

- [ ] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [x ] Have you discussed these changes with the project leader(s)?
- [x ] Do all variable and function names communicate what they do?
- [ x] Were all the changes commented and / or documented?
- [ ] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [ ] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [ ] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### After

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

File: frontend/src/components/CSVUploadModal/CSVUploadModal.scss [hunk 1]
```diff
@@ -106,16 +119,17 @@
 
     .modal-button {
       padding: 0.5em 2em;
-      border-radius: 6px;
-      border: 4px solid transparent;
-      background-color: $lightBronze;
+      border-radius: 8px;
+      // border: 4px solid transparent;
```
Comment (JMStudiosJoe) [inline frontend/src/components/CSVUploadModal/CSVUploadModal.scss:123]: Let's remove the commented out code please

File: frontend/src/components/CSVUploadModal/CSVUploadModal.scss [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
   .wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
     position: relative;
+    width: 100%;
 
     h2 {
-      display: inline-block;
+      display: flex;
```
Comment (JMStudiosJoe) [inline frontend/src/components/CSVUploadModal/CSVUploadModal.scss:19]: I'm not much of a fan of display flex but I know it has its uses in smaller areas. Have you used css grid before? But this use case does look like a small one.

File: frontend/src/components/LoginHandler/LoginHandler.scss [hunk 3]
```diff
@@ -76,7 +102,7 @@
                     letter-spacing:1.5px;
                     margin-left:1.5rem;
                     font-weight:600;
-                    font-size:1.5rem;
```
Comment (JMStudiosJoe) [inline frontend/src/components/LoginHandler/LoginHandler.scss:79]: Hmmm why going from rem to px? I thought best practices were to be moving from px to rem and vw?

**Review by JMStudiosJoe — CHANGES_REQUESTED** [review]
Just some small questions and some minor cleanup before merging... also only about half way done will review finish later.

File: frontend/src/components/CSVUploadModal/CSVUploadModal.scss (hunk 1 above)
Comment (gomezbiggeri) [reply frontend/src/components/CSVUploadModal/CSVUploadModal.scss]: removed commented line

File: frontend/src/components/CSVUploadModal/CSVUploadModal.scss (hunk 2 above)
Comment (gomezbiggeri) [reply frontend/src/components/CSVUploadModal/CSVUploadModal.scss:19]: I have, but not as much as flex, and I was having difficulty getting this one right.

File: frontend/src/components/LoginHandler/LoginHandler.scss (hunk 3 above)
Comment (gomezbiggeri) [reply frontend/src/components/LoginHandler/LoginHandler.scss:79]: I did this before learning about rem. I'll fix it.

**Review by gomezbiggeri — COMMENTED** [review]
I kept the updated loginContext function, and added my current design layout.

File: frontend/src/components/MeetingListView/MeetingListItem.jsx [hunk 4]
```diff
@@ -33,6 +37,9 @@ function MeetingListItem({ item }) {
   const date = toDateString(meeting_start_timestamp);
   const time = toTimeString(meeting_start_timestamp);
   const isInProgress = status === 'IN PROGRESS';
+  //LOCALDEV Hardcoded status for styling <----- REMOVE BEFORE COMMITING
```
Comment (JMStudiosJoe) [inline frontend/src/components/MeetingListView/MeetingListItem.jsx:40]: Remove before commiting?

File: frontend/src/components/MeetingListView/MeetingListItem.scss [hunk 5]
```diff
@@ -2,57 +2,86 @@
...
   and (min-device-width: 812px) {
-    grid-template-columns: 1.4em 2fr 1fr 1.4em;
-    grid-template-rows: 1.4em 1fr 1.4em 1fr 1.4em;
-    grid-gap: 0;
-    padding: 0;
-
-    h3 {
-      padding-top: 1.4em;
-      padding-left: 1.4em;
+    // grid-template-columns: 1.4em 2fr 2fr 1.4em;
```
Comment (JMStudiosJoe) [inline frontend/src/components/MeetingListView/MeetingListItem.scss:19]: can remove the commented out code as well here

File: frontend/src/components/MeetingListView/MeetingListItem.scss [hunk 6]
```diff
@@ -2,57 +2,86 @@
...
     }
 
+    
+
     .meeting-links {
-      grid-row: 4 / 5;
-      grid-column: 2 / 3;
+
+      // grid-row: 4 / 5;
+      // grid-column: 2 / 3;
```
Comment (JMStudiosJoe) [inline frontend/src/components/MeetingListView/MeetingListItem.scss:54]: these as well

**Review by JMStudiosJoe — COMMENTED** [review]
just some minor cleanup then good to go

**Pull request closed without merging**